presentation.tex

```diff
@@ -7,7 +7,7 @@
 
 \title{Vim vs. Emacs}
 \author{Brandon Moore and John Markiewicz}
-\date{}
```

Comment:
I switched it to use \date{} which should simply exclude date on the title page. I'm split on whether or not to include the date